app/services/idv/inherited_proofing/va/service.rb

```diff
@@ -17,12 +17,35 @@ def initialize(service_provider_data)
         def execute
           raise 'The provided auth_code is blank?' if auth_code.blank?
 
-          response = request
-          payload_to_hash decrypt_payload(response)
+          begin
+            response = request
+            return payload_to_hash decrypt_payload(response) if response.status == 200
+
+            service_error not_200_service_error(response.status)
+          rescue => error
+            return service_error error.message
```

Comment:
1. let's use parens for clarity
2. no need for the optional return
```suggestion
            service_error(not_200_service_error(response.status))
          rescue => error
            service_error(error.message)
```

Reply:
Consider enforcing this using rubocop perhaps?

Reply:
Addressed.